### PR TITLE
Update reference UI to be dynamic

### LINKED ISSUE #133: Update block and room reference in UI

### PR DESCRIPTION
**Pull request by schoolex**
image

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Block.java [hunk 1]
```diff
@@ -30,6 +30,10 @@ public Block(String block) {
         value = block.toUpperCase();
     }
 
+    public static List<String> getBlockList() {
+        return blockPref;
```
Comment (jieffyy) [inline src/main/java/seedu/address/model/person/Block.java]: Would you consider making this getter more defensive? e.g. return a copy/unmodifiable list.

File: src/main/java/seedu/address/model/person/Room.java [hunk 2]
```diff
@@ -43,9 +43,18 @@ public static boolean isValidRoom(String test) {
         return isFloorValid && isRoomValid;
     }
 
+    public static String getFLoorRange() {
```
Comment (jieffyy) [inline src/main/java/seedu/address/model/person/Room.java]: Is there a typo in the method name?

**Review by jieffyy — COMMENTED** [review]
Generally LGTM. A few simple areas for your consideration.

File: src/main/java/seedu/address/model/person/Room.java (hunk 2 above)
Comment (schoolex) [reply src/main/java/seedu/address/model/person/Room.java]: Good catch

**Review by jieffyy — APPROVED** [review]
LGTM

**Pull request merged**